### PR TITLE
Add variable description

### PR DESCRIPTION
**Pull request by nitinmahawadiwar**
Returning description field  for GET a)variables & b) variable API. 
SWAGGER documentation updated.
Test cases are incomplete, not able to understand the overall flow for test cases. will update them upon review.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
Looks cool, but I do not know a lot about updating schma version in this case should we do it @ephraimbuddy @mik-laj ?

File: airflow/api_connexion/schemas/variable_schema.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class VariableSchema(Schema):
 
     key = fields.String(required=True)
     value = fields.String(attribute="val", required=True)
-
+    description = fields.String(attribute="description", required=True)
```
Comment (ephraimbuddy) [inline airflow/api_connexion/schemas/variable_schema.py:26]: I don't think ``description`` should be a required field

File: airflow/api_connexion/endpoints/variable_endpoint.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def get_variable(*, variable_key: str) -> Response:
         var = Variable.get(variable_key)
     except KeyError:
         raise NotFound("Variable not found")
-    return variable_schema.dump({"key": variable_key, "val": var})
+    return variable_schema.dump({"key": variable_key, "val": var, "description": description})
```
Comment (ephraimbuddy) [inline airflow/api_connexion/endpoints/variable_endpoint.py:49]: I don't understand where the `description` here is coming from

**Review by ephraimbuddy — CHANGES_REQUESTED** [review]
`description` should not be a required field. It's a breaking change if it's a required field. So we should change it

File: airflow/api_connexion/schemas/variable_schema.py (hunk 1 above)
Comment (nitinmahawadiwar) [reply airflow/api_connexion/schemas/variable_schema.py:26]: thank you @ephraimbuddy for review & suggestion. 
Will update the changes for 'required' attribute & re-submit the PR.

**Comment by ephraimbuddy** [comment]
@nitinmahawadiwar , are you still working on this?

**Pull request closed without merging**